.github/workflows/algolia-search.yml

```diff
@@ -20,8 +20,10 @@ jobs:
     needs: [wait-for-vercel]
     steps:
       - uses: actions/checkout@v2
-      - uses: darrenjennings/algolia-docsearch-action@master
-        with:
-          algolia_application_id: ${{ secrets.ALGOLIA_APP_ID}}
-          algolia_api_key: ${{ secrets.ALGOLIA_ADMIN_API_KEY }}
-          file: "./algolia-config.json"
+      - name: Run algolia/docsearch-scraper
+        run: |
+          touch .env
+          echo ALGOLIA_APP_ID=${{ secrets.ALGOLIA_APP_ID }} >> .env
+          echo API_KEY=${{ secrets.ALGOLIA_ADMIN_API_KEY }} >> .env
```

Comment:
can probably just pass in the environment variables to the `algolia/docsearch-scraper` instead of creating a env file.